browser/rewards_data/counters/rewards_data_counter_utils.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/rewards_data/counters/rewards_data_counter_utils.h"
+
+#include "base/strings/utf_string_conversions.h"
```

Comment:
some obvious lint issues here with ordering so please ensure that lint is checked